### PR TITLE
Add method delete_remote_access

### PR DESCRIPTION
**Pull request by Sigabort42**
Add method delete_remote_access

Signed-off-by: el harag <elharag@cyberwatch.fr>

### CODE REVIEW

File: examples/delete_remote_access.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Delete remote access agentless"""
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = 'http://10.10.1.111'
```
Comment (jinxka) [inline examples/delete_remote_access.py]: remove url

File: examples/delete_remote_access.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+REMOTE_ID = 'Wrong_id'
+
+CLIENT.delete_remote_access(REMOTE_ID)
```
Comment (jinxka) [inline examples/delete_remote_access.py]: Add new line at the end of file

File: examples/delete_remote_access.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = 'http://10.10.1.111'
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+REMOTE_ID = 'Wrong_id'
```
Comment (jinxka) [inline examples/delete_remote_access.py]: remove `Wrong_id`

File: examples/delete_remote_access.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+"""Delete remote access agentless"""
```
Comment (fwininger) [inline examples/delete_remote_access.py]: remove agentless

File: cbw_api_toolbox/cbw_api.py [hunk 5]
```diff
@@ -106,3 +106,13 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def delete_remote_access(self, remote_id):
+        """DELETE request to /api/v2/remote_accesses to delete a specific remote access"""
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: /api/v2/remote_accesses/{remote_id}

File: cbw_api_toolbox/cbw_api.py [hunk 6]
```diff
@@ -106,3 +106,13 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def delete_remote_access(self, remote_id):
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: be consistent, remote_id => remote_access_id

File: cbw_api_toolbox/cbw_api.py [hunk 7]
```diff
@@ -106,3 +106,13 @@ def remote_accesses(self):
...
+        """DELETE request to /api/v2/remote_access/{remote_id} to delete a specific remote access"""
+        if remote_id:
+            print("Deleting remote access {}".format(remote_id))
+            response = self._request("DELETE", [ROUTE_REMOTE_ACCESSES, remote_id])
+            return self.verif_response(response)
+
+        print("No remote access id")
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: remote_access_id

File: examples/delete_remote_access.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+REMOTE_ID = ''
```
Comment (fwininger) [inline examples/delete_remote_access.py]: REMOTE_ACCESS_ID

File: cbw_api_toolbox/cbw_api.py (hunk 7 above)
Comment (Sigabort42) [reply cbw_api_toolbox/cbw_api.py]: :+1:

**Comment by fwininger** [comment]
@Sigabort42 rebase please

**Pull request merged**